packages/pnpm-workspace-yaml/src/index.test.ts

```diff
@@ -79,8 +83,11 @@ it('should work with anchor & alias', () => {
 
   expect(pw.toString()).toMatchInlineSnapshot(`
```

Comment:
Instead of super charging `setPackage`, I think we should have another entry, like `setPackageNoConficts` or so, as there are cases when people **do want** override the version. We should provide the permitives for them to do so.

Reply:
Okay, I'll do that later 👍

I think that the default for eslint should be with the `setPackageNoConficts`, I should update it also or ?

Reply:
We could have an option in eslint to toggle this behavior. Yeah we could have it enable by default

Reply:
Should better like this.